### ISSUE TITLE
Broken history recall layout after console resize with multiline prompt

### ISSUE THREAD

**Issue by nanoant** [PowerShell/PSReadLine#633]
I use multiline prompt, and everything works fine:

![image](https://user-images.githubusercontent.com/103067/36199799-e75b3422-117a-11e8-8158-937f1cf900f1.png)

Except when I resize my console window then the layout of the history (Up/Down arrow) gets broken after resize:

![image](https://user-images.githubusercontent.com/103067/36199814-f7681b78-117a-11e8-959d-1e4f09879049.png)

When advance to new command (via Enter) it gets recovered to "valid" state. It looks like readline is recovering the prompt using the "spaces" from the old window size, rather than new lines.

Here's a `profile.ps1` prompt setting that can be used to replicate the issue:
~~~powershell
function Prompt {
    Write-Host $($ExecutionContext.SessionState.Path.CurrentLocation) -NoNewline
    return "`n" + '>' * ($nestedPromptLevel + 1) + ' '
}
~~~

**Comment by lzybkr** [comment]
This should be fixed in [2.0.0-beta1](https://www.powershellgallery.com/packages/PSReadline/2.0.0-beta1), feel free to reopen with additional details if you update and can still repro.

**Comment by nanoant** [comment]
I confirm this is fixed in 2.0.0-beta1. Thank you!
